nuxt.config.js

```diff
@@ -22,7 +36,12 @@ module.exports = {
       }
     ]
   },
+  env,
+
+  publicRuntimeConfig:{
+    BASE_URL:env.BASE_URL,
 
+  },
```

Comment:
What is this? Why is it called `publicRuntimeConfig` and how is it different from `env` above? Also, please have a space in `:{`:

```suggestion
  publicRuntimeConfig: {
    BASE_URL:env.BASE_URL,
  },
```